pkgs/tools/wayland/kanshi/default.nix

```diff
@@ -8,25 +8,30 @@
 , wayland
 , wayland-scanner
 , libvarlink
+, libscfg
 }:
 
 stdenv.mkDerivation rec {
   pname = "kanshi";
-  version = "1.5.1";
+  version = "1.6.0";
 
   src = fetchFromSourcehut {
     owner = "~emersion";
     repo = "kanshi";
     rev = "v${version}";
-    sha256 = "sha256-Ck0yRt9TYLFRojn+VKnjP5RzkX0hciuQOT6drTH7gtU=";
+    sha256 = "sha256-KjP7EdssaZwa1OupLQgzwJSAADKLsjHltEavPjsS1YM=";
   };
 
   strictDeps = true;
   depsBuildBuild = [
     pkg-config
   ];
   nativeBuildInputs = [ meson ninja pkg-config scdoc wayland-scanner ];
-  buildInputs = [ wayland libvarlink ];
+  buildInputs = [ wayland libvarlink libscfg ];
+
+  env.NIX_CFLAGS_COMPILE = toString [
```

Comment:
There is probably a more standard way to do this, but it's fine

Reply:
Maybe I missed something but I only found this method when grepping nixpkgs :)

Reply:
You are fine. I think this is a good way of doing it, but generally people don't like directly setting environment variables in cases like this 🤷